packages/xo-server/src/xapi-stats.js

```diff
@@ -268,14 +284,10 @@ export default class XapiStats {
       .then(response => response.readAll().then(JSON5.parse))
   }
 
-  async _getNextTimestamp(xapi, host, step) {
+  async _getNextTimestamp(xapi, host, uuid, step) {
```

Comment:
I don't understand what this function does, can you find a better name or add a comment please?

Reply:
`getStartTimeStamp` or `getOptimumTimestamp` maybe ?  i need also to fix it to not use the last timestamp anymore because with this implementation we delete the cache.